### DISCUSSION TITLE
is there font cache in the program?

### DISCUSSION THREAD

**Discussion by ghiewa** [typst/typst#4102] [Q&A]
### Description

I run typst.exe on a windows platform. when font update in win os, it doesn't change in typst.  

### Operating system

Windows

### Typst version

- [X] I am using the latest version of Typst

**Comment by laurmaedje** [comment]
When you are running typst watch, you need to restart it. But there is no persistent cache, so if you've done that and it still doesn't pick the font up, something else is wrong.

**Reply by ghiewa** [reply]
How can I troubleshoot the specific bug? The issue is that the font update is not reflected in Typst (not using watch).

**Comment by alerque** [comment]
You almost certainly have more than one copy of the font installed, possibly in different locations, and the older one is in a location that is getting higher priority

**Reply by ghiewa** [reply]
obvious it is not. updated font installation works with other editor like ms word

**Reply by alerque** [reply]
Different font loaders might use different priorities, another app picking up the "new" one does not rule out there being more than one.

Also just to confirm before you move on, are you by chance trying to override one of the small collection of fonts that Typst embeds internally instead of using from the system?
